README.md

```diff
@@ -1,2 +1,72 @@
 # gnmi-python
-gNMI Python connectivity library. 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
+
+This library wraps gNMI functionality to ease usage with Cisco implementations in Python programs. Derived from [openconfig/gnmi](https://github.com/openconfig/gnmi/proto/gnmi/gnmi.proto).
+
+## Usage
+```bash
+# TBD
+#pip install gnmi
+```
+
+This library covers the gNMI defined `capabilities`, `get`, `set`, and `subscribe` RPCs, and helper clients provide OS-specific recommendations. As commonalities and differences are identified this library will be refactored as necessary.
+
+It is *highly* recommended that users of the library learn [Google Protocol Buffers](https://developers.google.com/protocol-buffers/) syntax to significantly ease usage. Understanding how to read Protocol Buffers, and reference the `gnmi.proto`, will be immensely useful for utilizing gNMI and any other gRPC interface.
```

Comment:
change "reference" to "referencing"

Reply:
Ooh we read it slightly differently. I was intending to keep this meaning separate. "referencing the `gnmi.proto`" in the context of the sentence would apply to "any other gRPC interface" which is untrue. The broader skill of reading protocol buffers applies to gNMI and any other gRPC interface, but referencing `gnmi.proto` does not.